service/src/test/java/org/databiosphere/workspacedataservice/service/ImportServiceTest.java

```diff
@@ -39,11 +52,30 @@ class ImportServiceTest {
 
   @Autowired ImportService importService;
   @Autowired InstanceService instanceService;
-  @Autowired JobDao jobDao;
+  @Autowired @SpyBean JobDao jobDao;
```

Comment:
nit: the `@Autowired` here is redundant (but harmless):
```suggestion
@SpyBean JobDao jobDao;
```